### ISSUE TITLE
Can't find the module

### ISSUE THREAD

**Issue by LorrdWealth** [sunilpaulmathew/NFSManager#7]
![Screenshot_2024-05-26-21-07-15-800_com.nfs.nfsmanager.jpg](https://github.com/sunilpaulmathew/NFSManager/assets/99515430/bfd71d23-be4f-4da6-8835-7aefffb0c016)

I have the correct module installed, but it says I don't have it. I have rebooted already.

**Comment by Chumra-max** [comment]
I have the same problem. Have you found a fix

**Comment by LorrdWealth** [comment]
> I have the same problem. Have you found a fix

Yeah I did, there's a group on telegram, I don't have access to it anymore, they game me a compatible app.. you can just search for NFC injector on telegram
